collective/trustedimports/suds.py

```diff
@@ -0,0 +1,33 @@
+from collective.trustedimports.util import whitelist_module
+import os
+from AccessControl import allow_class, ModuleSecurityInfo, ClassSecurityInfo, Unauthorized
+from AccessControl.class_init import InitializeClass
+from Products.PythonScripts.Utility import allow_module
+from suds.client import Client as _Client
+from suds import WebFault
+from suds import MethodNotFound
+
+
+class SafeClient(_Client):
+    """ SafeClient
+    """
+
+    def __init__(self, url, **kwargs):
+
+        blacklist = os.getenv('SAFEIMPORTS_URL_BLACKLIST ', '')
+        if url in blacklist:
+            raise Unauthorized("Not supported by SafeClient in this context")
```

Comment:
wrong kind of error. it shouldn't lead to a login. It should the the same error as when you do something else not allowed in restrictedpython like a.b = 1.